apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -44,31 +44,16 @@ export default class AnimationPickerBody extends React.Component {
     onPickLibraryAnimation: PropTypes.func.isRequired,
     onUploadClick: PropTypes.func.isRequired,
     playAnimations: PropTypes.bool.isRequired,
-    libraryManifest: PropTypes.object.isRequired,
+    getLibraryManifest: PropTypes.func.isRequired,
     categories: PropTypes.object.isRequired,
     hideUploadOption: PropTypes.bool.isRequired
   };
 
-  constructor(props) {
```

Comment:
We don't know for sure that the manifest has loaded here. We could either change the spritelab code to be async and wait, or be greedy and only call `searchAssets` when we need to search. I tested removing this (plus the changes below) and it worked fine for me but let me know if I missed a case here.

Reply:
very interesting! i may have unnecessarily refactored this code into what it is now. the expected/wanted behavior when this component loads is that it shows the "draw your own" option along with all of the available categories -- is that what currently happens?

Reply:
I think this looks right?

Spritelab:
![antimation-tab](https://user-images.githubusercontent.com/46464143/83186821-f4996200-a0e1-11ea-9a93-b9d2264caa42.gif)

Gamelab:
![gamelab-animation-tab](https://user-images.githubusercontent.com/46464143/83186990-4641ec80-a0e2-11ea-8155-79faf8525473.gif)

Reply:
it does! yay for deleting code 🎉 i wish i had noticed that when i refactored this constructor 😂